setup.py

```diff
@@ -10,6 +10,7 @@
     'cliff==2.4.0',
     'requests==2.13.0',
     'pytz==2017.2',
+    'pbr>=2.0.0',
```

Comment:
`pbr` is a dependency of `cliff`, which is one of our dependencies. But `pbr` shouldn't be a direct dependency for us.